tomviz/ImageStackModel.h

```diff
@@ -17,8 +17,9 @@
 #ifndef tomvizImageStackModel_h
 #define tomvizImageStackModel_h
 
-#include <QAbstractTableModel>
+#include "DataSource.h"
 
+#include <QAbstractTableModel>
```

Comment:
Generally always prefer including the header for what you derive from at the very top, then a blank line, then other stuff in grouped order.